### ISSUE TITLE
documentation on evolutionary models

### ISSUE THREAD

**Issue by SherelynA** [BDNYC/sedkit#142]
I feel like this info is somewhere, and if it is I think it should be easier to find what the references are for the evolutionary models. @hover2pi you probably have a better idea about if this is somewhere in the code.

**Comment by hover2pi** [comment]
It's just the references to the publications for now. Were you looking for something more like documentation? You can see the references used for a given SED by calling the `s.refs` attribute.

**Comment by SherelynA** [comment]
@hover2pi, yes it adds the evo model to the s.refs but it doesn't add the reference itself for the evo model to s.refs. So for example, it would say the bibcode for the evolutionary model being used.

**Comment by hover2pi** [comment]
@SherelynA Oh, we need to add that then. For the time being, the `nc_solar_age` evo model reference should be:

> D. Saumon and M. S. Marley. The Evolution of L and T Dwarfs in Color-Magnitude Di- agrams. Astrophysical Journal, 689(2):1327–1344, dec 2008. doi: 10.1086/592734. URL http://dx.doi.org/10.1086/592734.
